### PR TITLE
SCF-Session3 Assignment submittedd

### PR DESCRIPTION
**Pull request by meta-ramjilal-choudhary**

### CODE REVIEW

File: SCF/SCF-Session3/Quetion2/Marksheet.java [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+	
+	private double grades[];
+	
+	Marksheet(){}
+	
+	/*
+	 * @param  grades - Student grades array
+	 * @return nothing
+	 */
+	Marksheet(double grades[]){
```
Comment (metaamitasharma) [inline SCF/SCF-Session3/Quetion2/Marksheet.java]: double grades[] or double[] grades

File: SCF/SCF-Session3/Quetion2/Marksheet.java [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+	}
+	
+	/*
+	 * Compute Average of All grades of students
+	 * @param   nothing
+	 * @return  double - Average of all grades
+	 */
+	public double averageOfAllGrades() throws ArithmeticException{
+		
+		double totalGrades = 0.0;
```
Comment (metaamitasharma) [inline SCF/SCF-Session3/Quetion2/Marksheet.java]: gradesTotal makes sense instead of totalGrades

File: SCF/SCF-Session3/Quetion2/Marksheet.java [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+	 */
+	public double percentageOfStudentPassed() throws ArithmeticException {
+		int totalStudentPassed = 0;
+		for(int index=0; index < this.grades.length; index++) {
+			if (this.grades[index] >= 40.00) {
+				totalStudentPassed ++;
+			}
+		}
+		double percentage = (double)totalStudentPassed / (double)this.grades.length;
+		percentage *= 100.0;
```
Comment (metaamitasharma) [inline SCF/SCF-Session3/Quetion2/Marksheet.java]: why we are doing this in two statements ?

File: SCF/SCF-Session3/Quetion2/MainClassMarksheet.java [hunk 4]
```diff
@@ -0,0 +1,12 @@
+
+public class MainClassMarksheet extends Marksheet {
+	
+	public static void main(String args[]) {
+		double grades[] = {50.454, 44.6945, 60.6714, 50.3312, 40.2125, 10.2008};
+		Marksheet marksheet = new Marksheet(grades);
```
Comment (metaamitasharma) [inline SCF/SCF-Session3/Quetion2/MainClassMarksheet.java]: Show results along with proper text.

File: SCF/SCF-Session3/Quetion3/Area.java [hunk 5]
```diff
@@ -0,0 +1,43 @@
+import java.lang.ArithmeticException;
+
+public class Area {
```
Comment (metaamitasharma) [inline SCF/SCF-Session3/Quetion3/Area.java]: Missing implementation.

File: SCF/SCF-Session3/Quetion1/MyString.java [hunk 6]
```diff
@@ -0,0 +1,132 @@
...
+	 * @return  length of string (int)
+	 */
+	public int strlen(String string) {
+		int index = 0;
+		try {
+			while(index > -1) {
+				string.charAt(index);
+				index++;
+			}
+			return index;
```
[the remark this answers is not in the document]
Comment (meta-ramjilal-choudhary) [reply SCF/SCF-Session3/Quetion1/MyString.java]: No, 
But in Try{} blick we should return something otherwise it throw an error

**Pull request merged**